### PR TITLE
Add gcloud marketplace self-signed certificate linux instruction

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Add gcloud marketplace self-signed certificate linux instruction.

**Related issue(s)**
See also: https://github.com/kyma-incubator/marketplaces/issues/34

### CODE REVIEW

**Review by hudymi — APPROVED** [review]

**Comment by kazydek** [comment]
See the reviewed instruction below:
[Uploading linux-marketplace.md.zip…]()

**Review by kazydek — COMMENTED** [review]
See the reviewed instruction below:
[linux-marketplace.md.zip](https://github.com/kyma-incubator/marketplaces/files/3845253/linux-marketplace.md.zip)

File: google-cloud/manifest/application.yaml [hunk 1]
```diff
@@ -27,32 +27,83 @@ spec:
         url: https://github.com/kyma-project
     notes: |-
       To access your Kyma cluster, import the Kyma self-signed certificate to your trusted certificate authorities.
-
       ### MacOS
-
       Copy the following command and paste it into your terminal window:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:31]:
```suggestion
      1. Copy the following command and paste it into your terminal window:
```

File: google-cloud/manifest/application.yaml [hunk 2]
```diff
@@ -27,32 +27,83 @@ spec:
...
-
       ```bash
       tmpfile=$(mktemp /tmp/temp-cert.XXXXXX) \
           && pbpaste > $tmpfile \
           && sudo security add-trusted-cert -d -r trustRoot -k /Library/Keychains/System.keychain $tmpfile \
           && rm $tmpfile
       ```
-
       On the GCP website, click **preview secret data** next to the **Certificate** details, and copy the certificate string.
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:38]:
```suggestion
      2. On the GCP website, click **preview secret data** next to the **Certificate** details, and copy the certificate string.
```

File: google-cloud/manifest/application.yaml [hunk 3]
```diff
@@ -27,32 +27,83 @@ spec:
...
       
       Run the command you previously pasted into your terminal window.
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:40]:
```suggestion
      3. Run the command you previously pasted into your terminal window.
```

File: google-cloud/manifest/application.yaml [hunk 4]
```diff
@@ -27,32 +27,83 @@ spec:
...
 
-      Run the Command Prompt as an Administrator. Copy the following command and paste it into your Command Prompt window:
+      ```
+      export KYMA_CERT=/tmp/kyma.crt
+      ```
+      Choose your Linux distribution and follow next steps.
+
+      #### Debian-based distro
+
+      Check if the `ca-certificates` package is installed:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:52]:
```suggestion
      1. Check if the `ca-certificates` package is installed:
```

File: google-cloud/manifest/application.yaml [hunk 5]
```diff
@@ -27,32 +27,83 @@ spec:
...
 
       ```
-      powershell "$(Get-Clipboard | Out-File -FilePath kyma.cer | Out-Host;$?) -and $(certutil -f -addstore Root kyma.cer | Out-Host;$?) -and $(Remove-Item -path kyma.cer | Out-Host;$?)"
+      apt list ca-certificates
       ```
+      If the package is not listed, install it with this command:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:57]:
```suggestion
      2. If the package is not listed, install it with this command:
```

File: google-cloud/manifest/application.yaml [hunk 6]
```diff
@@ -27,32 +27,83 @@ spec:
...
 
-      On the GCP website, click **preview secret data** next to the **Certificate** details, and copy the certificate string.
+      ```
+      sudo apt install ca-certificates
+      ```
+      Copy the Kyma certificate and update the certificate list:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:62]:
```suggestion
      3. Copy the Kyma certificate and update the certificate list:
```

File: google-cloud/manifest/application.yaml [hunk 7]
```diff
@@ -27,32 +27,83 @@ spec:
...
 
+      Run this command:
+
+      ```
+      sudo sh -c "cp ${KYMA_CERT} /etc/ca-certificates/trust-source/anchors && \
+      trust extract-compat && rm -i ${KYMA_CERT}"
+      ```
+      #### Centos6
+      ​
+      Check if the `ca-certificates` package is installed:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:78]:
```suggestion
      1. Check if the `ca-certificates` package is installed:
```

File: google-cloud/manifest/application.yaml [hunk 8]
```diff
@@ -27,32 +27,83 @@ spec:
...
+      ​
+      ```
+      yum list installed ca-certificates
+      ```
+      If the package is not listed, install it with this command:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:83]:
```suggestion
      2. If the package is not listed, install it with this command:
```

File: google-cloud/manifest/application.yaml [hunk 9]
```diff
@@ -27,32 +27,83 @@ spec:
...
+
+      ```
+      yum install ca-certificates
+      ```
+      Copy the Kyma certificate and update the certificate list:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:88]:
```suggestion
      3. Copy the Kyma certificate and update the certificate list:
```

File: google-cloud/manifest/application.yaml [hunk 10]
```diff
@@ -27,32 +27,83 @@ spec:
...
+      ```
+      #### Centos5
+      ​
+      Run this command:
+      ​
+      ```
+      sudo sh -c "cat ${KYMA_CERT} >>/etc/pki/tls/certs/ca-bundle.crt"
+      ```              
+      ### Windows
+      Run the Command Prompt as an Administrator. Copy the following command and paste it into your Command Prompt window:
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml]:
```suggestion
      1. Run the Command Prompt as an Administrator. 
      
      2. Copy the following command and paste it into your Command Prompt window:
```

File: google-cloud/manifest/application.yaml [hunk 11]
```diff
@@ -27,32 +27,83 @@ spec:
...
+      ```
+      powershell "$(Get-Clipboard | Out-File -FilePath kyma.cer | Out-Host;$?) -and $(certutil -f -addstore Root kyma.cer | Out-Host;$?) -and $(Remove-Item -path kyma.cer | Out-Host;$?)"
+      ```
+      On the GCP website, click **preview secret data** next to the **Certificate** details, and copy the certificate string.
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:108]:
```suggestion
      3. On the GCP website, click **preview secret data** next to the **Certificate** details, and copy the certificate string.
```

File: google-cloud/manifest/application.yaml [hunk 12]
```diff
@@ -27,32 +27,83 @@ spec:
...
       Run the command you previously pasted into your Command Prompt window.
```
Comment (kazydek) [inline google-cloud/manifest/application.yaml:109]:
```suggestion
      4. Run the command you previously pasted into your Command Prompt window.
```

**Review by kazydek — APPROVED** [review]

**Comment by kazydek** [comment]
As discussed, pls use the version attached, thx :)

**Pull request merged**